### ISSUE TITLE
Word highlighting and recitation are not in synchronization.

### ISSUE THREAD

**Issue by fahad-ibrar** [quran/quran.com-frontend#986]
**I'm submitting a ...**

- [x] bug report
- [ ] feature request
- [ ] other

**Current behavior:**

There is an issue in the player. While playing the recitation the word being recited is not highlighted. There is a difference between the time at which word highlighted and recited. The recitation is quicker than the highlighting. It usually lags for 2-3 words.

**Expected / new behavior:**

Recitation should be in sync with word highlighting. When a word is recited it should be highlighted at the same time.

**Comment by mrtugicom** [comment]
Yeah, me too some times after playing the audio for a while, it happens.
